doc/ale.txt

```diff
@@ -1234,6 +1234,17 @@ g:ale_floating_preview                                 *g:ale_floating_preview*
   |g:ale_detail_to_floating_preview| to `1`.
 
 
+g:ale_floating_preview_popup_opts             *g:ale_floating_preview_popup_opts*
+
+  Type: |String|
+  Default: `''`
+
```

Comment:
Make clear that his option takes effect only in vim and not neovim.  Also if we are going to support both funcref and dict update the type too.